### DISCUSSION TITLE
til-site/posts/one-problem-with-user-scoped-systemd-timers/

### DISCUSSION THREAD

**Discussion by giscus** [hiAndrewQuinn/til#9] [Announcements]
# til-site/posts/one-problem-with-user-scoped-systemd-timers/

I’ve discovered one big downside of [putting systemd times into ~/.config/systemd/user/][1]: They stop running when you log out.
Two ways around this problem:
Bite the bullet, and install the timers into /etc/systemd/. This means giving up chezmoi version control, making the overall system more snowflake-y. I’m not crazy about it. Remote in with tmux, instead of ever logging out, just use C-b d to detach from the session. I’m going with #2.

https://hiandrewquinn.github.io/til-site/posts/one-problem-with-user-scoped-systemd-timers/

**Comment by hiAndrewQuinn** [comment]
Turns out there's a third way around this problem: Just use `loginctl enable-linger`. 🤦‍♂️
